src/common/colorspaces.c

```diff
@@ -2454,12 +2450,10 @@ void dt_colorspaces_rgb_to_cygm(float *out,
                                 const int num,
                                 const double RGB_to_CAM[4][3])
 {
-#ifdef _OPENMP
-#pragma omp parallel for default(none) shared(out, num, RGB_to_CAM) schedule(static)
-#endif
+  DT_OMP_FOR()
   for(int i = 0; i < num; i++)
   {
-    float *in = &out[i*3];
+    float *in = &out[i*3];  //FIXME: is this correct or should it be i*4 ?
```

Comment:
Difficult to test as only module using this (invert) is depricated and i have only one of those images.